docs/zh-tw/translation-guide.md

```diff
@@ -6,24 +6,20 @@
 
 ## 翻譯標題 ID
 
-基於自動產生文章內的導覽、識別程式碼區塊用於建立實際範例、以及其他的原因，我們的文章通常都會為標題配上 ID 。
-
-在翻譯標題時，你不需要翻譯它的 ID。路徑的其他部分也沒有翻譯，所以讓它維持一致吧。
+基於自動產生文章內的導覽、識別程式碼區塊用於建立實際範例、以及其他的原因，我們的文章會使用到 HTML ID。而對於 Markdown 文件，其渲染得到的 HTML 文件的 ID 則與 Markdown 文件中的標題相對應（但這並不意味著兩者相同，yari 平台對於空格、半角標點等有特殊處理）。
```

Comment:
We'd better to add `how to get the html ID` here.

@irvin, could you help to add this. You can refer this doc: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Live_samples#block_id

Reply:
I've added this part.